pom.xml

```diff
@@ -54,11 +54,6 @@
             <artifactId>jakarta.data-api</artifactId>
             <version>1.0.0</version>
         </dependency>
```

Comment:
We should probably manage this dependency in Quarkus. https://github.com/quarkusio/quarkus/issues/44278#issuecomment-2459079583